Dockerfile.builder

```diff
@@ -2,7 +2,7 @@
 # (and committing your edits) you should run ./push-builder.sh to
 # push a new version of your image to quay.io/coreos/tectonic-console-builder
 
-FROM golang:1.11-stretch
+FROM golang:1.12
```

Comment:
You should remove this from the PR. It doesn't change what's used in CI and will conflict with #3992

Reply:
Right, this looks like a leftover from some ci experiments. 

Akash is on it

Reply:
Reverted this changes